doc/manifest-tool-guide.md

```diff
@@ -0,0 +1,92 @@
+# Manifest Tool Guide
+
+This guide explains how to setup a validator so the key pairs used to sign and
+verify validations may safely change. This procedure does not require manual
+reconfiguration of servers that trust this validator.
+
+Validators use two types of key pairs: *master keys* and *ephemeral
+keys*. Ephemeral keys are used to sign and verify validations. Master keys are
+used to sign and verify manifests that change ephemeral keys. The master secret
+key should be tightly controlled. The ephemeral secret key needs to be present
+in the config file.
+
+## Validator Keys
+
+When first setting up a validator, use the `validator-keys` tool to generate a
+master key pair:
+
+```
+  $ validator-keys create_master_keys
+```
+
+Sample output:
+```
+  Master validator keys stored in /home/ubuntu/.ripple/validator-keys.json
+```
+
+Sample key file:
+```
+  {
+     "key_type" : "ed25519",
+     "master_secret" : "paUvjG1z51cpSxL3Wc9DaGTEogYk4AEesLJJaawBcVQjejxGFcb",
+     "sequence" : 4294967295,
+     "validation_public_key" : "nHUtNnLVx7odrz5dnfb2xp1gbEeJPbzJWfdicSkGyVw1eE5GpjQr"
+  }
+```
+
+The `validation_public_key` is the master public key. Any other rippled trusting
+the validator needs to add the master public key to its config. Only add keys
+received from trusted sources.
+
+The `master_secret` is the corresponding master secret key. **DO NOT SHARE THIS
+KEY**. The master secret key will be used to sign manifests that change
+ephemeral validation keys. Put the master secret key in a secure but recoverable
+location.
+
+## Validation Keys
+
+When first setting up a validator, or when changing the ephemeral keys, use the
+`validator-keys` tool to create a new ephemeral key pair:
+
+```
+  $ validator-keys create_signing_keys
```

Comment:
I agree that this is a good opportunity to improve the names.

What do you think about this?

A validator has two sets of keys: master keys and validation keys used to sign validations.
We keep the program name as `validator-keys` but rename the `create_signing_keys` command to `create_validation_keys`.
`validator-keys.json` could keep its name or change to `validator-master-keys.json`. The `validation_public_key` field in the file can be changed to `master_public_key`.